File: VB/HeckYaBaby/Backend/Domain/ProfileEntity.vb
Namespace Domain
    Public Class ProfileEntity 
        Implements IAuditable

        Public Property Id As Integer
        Public Property Address As String
        Public Property DateOfBirth As Date
        Public Property FriendCount As Integer
        Public Property Name As String
        Public Property PhoneNumber As String
        Public Property RowDate As Date Implements IAuditable.RowDate
        Public Property RowStatus As String Implements IAuditable.RowStatus

    End Class
End NameSpace

File: C#/HeckYaBaby/Service/Domain/ProfileEntity.cs
using System;

namespace Service.Domain
{
    public class ProfileEntity : IAuditable
    {
        public int Id { get; private set; }
        public string Name { get; set; }
        public string PhoneNumber { get; set; }        
        public string Address { get; set; }
        public DateTime DateOfBirth { get; set; }
        public int FriendCount { get; set; }
        public DateTime RowDate { get; set; }
        public string RowStatus { get; set; }
    }
}